### PR TITLE
Mission Support System recipe

### PR DESCRIPTION
**Pull request by ReimarBauer**

### CODE REVIEW

File: recipes/mss/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+{% set version = "1.1.0" %}
+
+
+package:
+  name: mss
+  version: {{ version }}
+
+source:
+    git_url: https://bitbucket.org/wxmetvis/mss.git
+    git_rev: 1.1.0
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: Please download a tarball instead.

```
fn: mss-{{ version }}.tar.gz
url: https://bitbucket.org/wxmetvis/mss/get/{{ version }}.tar.gz
sha256: <some sha here>
```

File: recipes/mss/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+build:
+    number: 0
+    script: python setup.py install --record record.txt
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: `python setup.py install` should be fine unless there is another reason for this.

File: recipes/mss/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    entry_points:
+      - mss = msui.mss_pyui:main
+
+requirements:
+  build:
+    - basemap ==1.0.7
+    - chameleon >=2.24
+    - curl ==7.26
+    - cycler ==0.10.0
+    - execnet ==1.2.0
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: These constraints seem kind of tight. Why is this? `curl` has a very stable ABI over the long range. The other two are Python and don't really need such tight constraints.

File: recipes/mss/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    - freetype >=2.5.0
+    - functools32 >=3.2.3.2
+    - geopy >=1.10.0
+    - hdf5 >=1.8.15
+    - jpeg >=8d
+    - libpng >=1.5.0
+    - lxml ==3.6.0
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: Also, pure Python. Why is the constraint exact?

File: recipes/mss/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+    - matplotlib >=1.4.0
+    - netcdf4 >=1.0
+    - numpy ==1.10.4
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: This should not have a version specified here.

File: recipes/mss/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+    - paste >=1.7.5
+    - pillow >=3.2.0
+    - pyqt >=4.11.4
+    - python ==2.7.11
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: This should not have a version specified here.

File: recipes/mss/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+    - freetype >=2.5.0
+    - functools32 >=3.2.3.2
+    - geopy >=1.10.0
+    - hdf5 >=1.8.15
+    - jpeg >=8d
+    - libpng >=1.5.0
+    - lxml ==3.6.0
+    - matplotlib >=1.4.0
+    - netcdf4 >=1.0
+    - numpy ==1.10.4
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: This should not have a version specified here.

File: recipes/mss/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+    - paste >=1.7.5
+    - pillow >=3.2.0
+    - pyqt >=4.11.4
+    - python ==2.7.11
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: This should not have a version specified here.

File: recipes/mss/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+    - python-dateutil ==2.5.2
+    - pytz ==2016.3
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: Again these are way too tight. Please use a range or lower bound.

File: recipes/mss/meta.yaml [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+    - qt >=4.8.6
+    - requests ==2.9.1
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: Same story too tight.

File: recipes/mss/meta.yaml [hunk 11]
```diff
@@ -0,0 +1,90 @@
...
+    - scipy >=0.17.0
+    - sip ==4.16.9
+    - six ==1.10.0
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: Too tight.

File: recipes/mss/meta.yaml [hunk 12]
```diff
@@ -0,0 +1,90 @@
...
+
+build:
+    number: 0
+    script: python setup.py install --record record.txt
+    entry_points:
+      - mss = msui.mss_pyui:main
+
+requirements:
+  build:
+    - basemap ==1.0.7
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: Too tight.

File: recipes/mss/meta.yaml [hunk 13]
```diff
@@ -0,0 +1,90 @@
...
+    - python-dateutil ==2.5.2
+    - pytz ==2016.3
+    - qt >=4.8.6
+    - requests ==2.9.1
+    - scipy >=0.17.0
+    - sip ==4.16.9
+    - six ==1.10.0
+    - tk >=8.5.18
+  run:
+    - basemap ==1.0.7
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: Too tight.

File: recipes/mss/meta.yaml [hunk 14]
```diff
@@ -0,0 +1,90 @@
...
+    - chameleon >=2.24
+    - curl ==7.26
+    - cycler ==0.10.0
+    - execnet ==1.2.0
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: Too tight.

File: recipes/mss/meta.yaml [hunk 15]
```diff
@@ -0,0 +1,90 @@
...
+    - lxml ==3.6.0
+    - matplotlib >=1.4.0
+    - netcdf4 >=1.0
+    - numpy ==1.10.4
+    - paste >=1.7.5
+    - pillow >=3.2.0
+    - pyqt >=4.11.4
+    - python ==2.7.11
+    - python-dateutil ==2.5.2
+    - pytz ==2016.3
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: Too tight.

File: recipes/mss/meta.yaml [hunk 16]
```diff
@@ -0,0 +1,90 @@
...
+    - qt >=4.8.6
+    - requests ==2.9.1
+    - scipy >=0.17.0
+    - sip ==4.16.9
+    - six ==1.10.0
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: Too tight.

File: recipes/mss/meta.yaml [hunk 17]
```diff
@@ -0,0 +1,90 @@
...
+    - netcdf4 >=1.0
+    - numpy ==1.10.4
+    - paste >=1.7.5
+    - pillow >=3.2.0
+    - pyqt >=4.11.4
+    - python ==2.7.11
+    - python-dateutil ==2.5.2
+    - pytz ==2016.3
+    - qt >=4.8.6
+    - requests ==2.9.1
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: Too tight.

File: recipes/mss/meta.yaml [hunk 18]
```diff
@@ -0,0 +1,90 @@
...
+about:
+  home: http://mss.rtfd.io
+  license: APACHE 2.0
+  license_file: LICENSE
+  summary: A web service based tool to plan atmospheric research flights
+
+extra:
+  recipe-maintainers:
+    - ReimarBauer
+
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: nit: drop extra newline. one terminal newline is sufficient.

File: recipes/mss/meta.yaml [hunk 19]
```diff
@@ -0,0 +1,90 @@
...
+    - tk >=8.5.18
+
+
+test:
+   imports:
+      - mslib
+
+about:
+  home: http://mss.rtfd.io
+  license: APACHE 2.0
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: nit: APACHE -> Apache

File: recipes/mss/meta.yaml [hunk 20]
```diff
@@ -0,0 +1,90 @@
...
+    - basemap ==1.0.7
+    - chameleon >=2.24
+    - curl ==7.26
+    - cycler ==0.10.0
+    - execnet ==1.2.0
+    - freetype >=2.5.0
+    - functools32 >=3.2.3.2
+    - geopy >=1.10.0
+    - hdf5 >=1.8.15
+    - jpeg >=8d
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: These need to match our [pinned version]( https://github.com/conda-forge/conda-forge.github.io/blob/352aab2876ff1dd4ad0163c67384f29b5e56646d/scripts/pin_the_slow_way.py#L32-L48 ).

File: recipes/mss/meta.yaml [hunk 21]
```diff
@@ -0,0 +1,90 @@
...
+    - libpng >=1.5.0
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: These need to match our [pinned version]( https://github.com/conda-forge/conda-forge.github.io/blob/352aab2876ff1dd4ad0163c67384f29b5e56646d/scripts/pin_the_slow_way.py#L32-L48 ).

File: recipes/mss/meta.yaml [hunk 22]
```diff
@@ -0,0 +1,90 @@
...
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: These need to match our [pinned version]( https://github.com/conda-forge/conda-forge.github.io/blob/352aab2876ff1dd4ad0163c67384f29b5e56646d/scripts/pin_the_slow_way.py#L32-L48 ).

File: recipes/mss/meta.yaml [hunk 23]
```diff
@@ -0,0 +1,90 @@
...
+    - lxml ==3.6.0
+    - matplotlib >=1.4.0
+    - netcdf4 >=1.0
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: These need to match our [pinned version]( https://github.com/conda-forge/conda-forge.github.io/blob/352aab2876ff1dd4ad0163c67384f29b5e56646d/scripts/pin_the_slow_way.py#L32-L48 ).

File: recipes/mss/meta.yaml [hunk 24]
```diff
@@ -0,0 +1,90 @@
...
```
Comment (jakirkham) [inline recipes/mss/meta.yaml]: These need to match our [pinned version]( https://github.com/conda-forge/conda-forge.github.io/blob/352aab2876ff1dd4ad0163c67384f29b5e56646d/scripts/pin_the_slow_way.py#L32-L48 ).

**Comment by jakirkham** [comment]
I have taken the liberty of renaming this to be a little clearer. I hope that is ok, @ReimarBauer.

**Comment by jakirkham** [comment]
So, many of the same points still stand, @ReimarBauer. There are lots of pinnings here that are too tight. If you could please fix those, that would go a long ways towards getting this ready to go. Also, we prefer not to clone the source, but instead prefer to download tarballs. Could you please update this accordingly?

File: recipes/mss/meta.yaml (hunk 18 above)
Comment (ReimarBauer) [inline recipes/mss/meta.yaml]: seen your note, will shortly look into this

**Comment by ReimarBauer** [comment]
seen will fix. Thx :)

File: recipes/mss/meta.yaml (hunk 3 above)
Comment (ReimarBauer) [inline recipes/mss/meta.yaml]: the problem is that netCDF4 has a dependency to curl. Newer curls have removed SSLv2 symbols.

File: recipes/mss/meta.yaml (hunk 4 above)
Comment (ReimarBauer) [inline recipes/mss/meta.yaml]: We have had verified that those works on windows and linux

File: recipes/mss/meta.yaml [hunk 25]
```diff
@@ -0,0 +1,90 @@
+{% set version = "1.1.0" %}
+
+
+package:
+  name: mss
+  version: {{ version }}
+
+source:
+    git_url: https://bitbucket.org/wxmetvis/mss.git
```
Comment (ocefpaf) [inline recipes/mss/meta.yaml]: @ReimarBauer what we need here is to user a tarball instead of the git repository to avoid a huge download. Here is how this section should look like:

```yaml
source:
  fn: mss-{{ version }}.tar.gz
  url: http://bitbucket.org/wxmetvis/mss/get/{{ version }}.tar.gz
  sha256: 0529bf9e41b878de4b82278c52469cd88d95531e4e4296bac71f7f44f6d3f93b
```

You can copy and paste that over your `source` section above.

File: recipes/mss/meta.yaml (hunk 3 above)
Comment (ocefpaf) [inline recipes/mss/meta.yaml]: Can you check if the `curl` present in `conda-forge` does have what you need? We are fine pinning software in a case like that, but you are pinning many of them, like `cycler` that is pure Python, and we believe you don't really need that.

Also, see to keep the stack consistent, try to follow `conda-forge`'s pinnings for those that might have an `ABI` change in different version. See https://github.com/conda-forge/staged-recipes/wiki/Pinned-dependencies

Comment (ReimarBauer) [inline recipes/mss/meta.yaml]: You are right, currently trying ;)
 At the moment I have a simpler test for that curl issue it would be much easier.

The pinning was done because these versions were verified that the whole package besides the ssl issue runs on windows and linux.

There were some packages which created issues, e.g. basemap-1.0.8  on client and server but some only on server and only in TLS (ssl) setup

Comment (ocefpaf) [inline recipes/mss/meta.yaml]: I can help you try to investigate those issues. I believe `basemap-1.0.8` is not to blame, but something of its dependencies might be. Can you post the errors you are seeing? (Or point me to any CI log?)

PS: Let's try to get most of the safe pins (pure Python for example) loose and get the other pins consisted with our [recommendation](https://github.com/conda-forge/staged-recipes/wiki/Pinned-dependencies). Then we can start debugging.

File: recipes/mss/meta.yaml (hunk 25 above)
Comment (ReimarBauer) [inline recipes/mss/meta.yaml]: hmm,  my sha256 looks different 
sha256: 6e15a2a7c65ce676169551660e51ced5b0718ef83c4f93104a0f6b70ed502183

I tried 
wget http://bitbucket.org/wxmetvis/mss/get/1.2.0.tar.gz
sha256sum  1.2.0.tar.gz

Comment (ocefpaf) [inline recipes/mss/meta.yaml]: It should :smile: I computed the checksum for version `1.1.0`. If you are updating it to `1.2.0` you have to update the `sha256` too. (Sorry, that is an extra step, but ensures that the tarball is correct.)

File: recipes/mss/meta.yaml (hunk 3 above)
Comment (ReimarBauer) [inline recipes/mss/meta.yaml]: on windows with version pinning  >= I find this pyproj issue. I don't have with basemap-1.0.7.

I check next minutes the server code.

```
  File "C:\Users\user\Miniconda2\envs\mssenv3\Scripts\mss-script.py", line 3, in <module>
    import mslib.msui.mss_pyui
  File "C:\Users\user\Miniconda2\envs\mssenv3\lib\site-packages\mslib\msui\mss_pyui.py", line 59, in <module>
    from mslib.msui import topview
  File "C:\Users\user\Miniconda2\envs\mssenv3\lib\site-packages\mslib\msui\topview.py", line 47, in <module>
    from mslib.msui import ui_topview_window as ui
  File "C:\Users\user\Miniconda2\envs\mssenv3\lib\site-packages\mslib\msui\ui_topview_window.py", line 135, in <module>
    from mpl_qtwidget import MplTopViewWidget
  File "C:\Users\user\Miniconda2\envs\mssenv3\lib\site-packages\mslib\msui\mpl_qtwidget.py", line 42, in <module>
    from mslib.msui import mpl_map
  File "C:\Users\user\Miniconda2\envs\mssenv3\lib\site-packages\mslib\msui\mpl_map.py", line 48, in <module>
    import mpl_toolkits.basemap.pyproj as pyproj
ImportError: No module named pyproj
```

```
# packages in environment at C:\Users\user\Miniconda2\envs\mssenv3:
#
chameleon                 2.24                     py27_0  
basemap                   1.0.8.dev0          np111py27_1    conda-forge
ca-certificates           2016.2.28                     1    conda-forge
curl                      7.48.0                    vc9_2  [vc9]  conda-forge
cycler                    0.10.0                   py27_0    conda-forge
functools32               3.2.3.2                  py27_1    conda-forge
geos                      3.4.2                     vc9_2  [vc9]  conda-forge
hdf4                      4.2.11                    vc9_0  [vc9]  conda-forge
hdf5                      1.8.17                    vc9_2  [vc9]  conda-forge
libnetcdf                 4.4.0                     vc9_1  [vc9]  conda-forge
libpng                    1.6.23                    vc9_0  [vc9]  conda-forge
netcdf4                   1.2.4               np111py27_2    conda-forge
openssl                   1.0.2h                    vc9_0  [vc9]  conda-forge
paste                     2.0.3                    py27_0    conda-forge
pillow                    3.2.0                    py27_0    conda-forge
pyparsing                 2.1.1                    py27_0    conda-forge
pyproj                    1.9.5.1                  py27_0    conda-forge
pyshp                     1.2.3                    py27_0    conda-forge
requests                  2.10.0                   py27_0    conda-forge
setuptools                23.0.0                   py27_0    conda-forge
sip                       4.18                     py27_0    conda-forge
six                       1.10.0                   py27_0    conda-forge
tk                        8.5.19                    vc9_0  [vc9]  conda-forge
zlib                      1.2.8                     vc9_3  [vc9]  conda-forge
execnet                   1.3.0                    py27_1  
freetype                  2.5.5                     vc9_1  [vc9]
jpeg                      8d                        vc9_0  [vc9]
libtiff                   4.0.6                     vc9_2  [vc9]
mss                       1.2.0                         0    local
lxml                      3.6.0                    py27_0  
matplotlib                1.5.1               np111py27_0  
mkl                       11.3.3                        1  
numpy                     1.11.1                   py27_0  
pip                       8.1.2                    py27_0  
pyqt                      4.11.4                   py27_6  
python                    2.7.12                        0  
python-dateutil           2.5.3                    py27_0  
pytz                      2016.4                   py27_0  
qt                        4.8.7                     vc9_8  [vc9]
scipy                     0.17.1              np111py27_1  
vs2008_runtime            9.00.30729.1                  2  
wheel                     0.29.0                   py27_0  

```

Comment (ocefpaf) [inline recipes/mss/meta.yaml]: Not sure what is going on in your build env, but I am sure we have `pyproj` in our `basemap 1.0.8`. See https://github.com/conda-forge/basemap-feedstock/blob/master/recipe/meta.yaml#L24

Comment (ocefpaf) [inline recipes/mss/meta.yaml]: @ReimarBauer just unpin it and commit. Let's see how AppVeyor handles this.

Comment (ReimarBauer) [inline recipes/mss/meta.yaml]: recent curl 7.4.9 needs netCDF1.2.4 and that changes to this issue
```
import mss_wms_settings
File "/home/mss/config/mss_wms_settings.py", line 56, in <module>
import netCDF4
File "/home/mss/miniconda2/lib/python2.7/site-packages/netCDF4/__init__.py", line 3, in <module>
from ._netCDF4 import *
libmfhdf.so.0: cannot open shared object file: No such file or directory
```

Comment (ocefpaf) [inline recipes/mss/meta.yaml]: Reimar it is hard to follow errors on your local machine. If you commit
those tests we can see the logs in the CIs and debug it. In this particular
error, for example, I don't know if were your netCDF4 1.2.4 is from. Also,
the dependence chain is netCDF4 -> libnetcdf -> curl. That may give us a
clue to what needs fixing/updating as curl is at the bottom of the stackand
installing modern versions of any software will cause it to upgrade.


**********************************************************
Filipe P. A. Fernandes
Physical Oceanographer

Email: ocefpaf@gmail.com

http://ocefpaf.github.io/
**********************************************************

**Comment by ReimarBauer** [comment]
next problem is to find the upper limit of packages for pinning.

if I do now

`from mslib.msui import ui_mainwindow as ui`

I get
```
from PyQt4 import QtCore, QtGui
ImportError: libicui18n.so.56: cannot open shared object file: No such file or directory
```
pyqt                      4.11.4 
qt                        4.8.7

**Comment by ocefpaf** [comment]
@ReimarBauer the `qt`/`icu` issue is not your fault and we are addressing it. Should be fixed soon...

File: recipes/mss/meta.yaml (hunk 3 above)
Comment (ReimarBauer) [inline recipes/mss/meta.yaml]: @ocefpaf 

So I think I found the difference.for basemap. On some place of my code I missed to capsule like

```
try:
    import mpl_toolkits.basemap.pyproj as pyproj
except ImportError:
    import pyproj
```
additional for same functionality I have had to add manually

`conda install -c conda-forge basemap-data-hires`

without that package it crashes. Maybe that could be added as a dependency? Because it seems not needed for 1.0.7.

Comment (ocefpaf) [inline recipes/mss/meta.yaml]: > On some place of my code I missed to capsule

Makes sense.

> Maybe that could be added as a dependency? Because it seems not needed for 1.0.7.

That is a design decision decision where `conda-forge` is different from `defaults`. We don't ship all the data b/c it is a heavy download, and not always needed. Since you clearly need it you have to install both packages (basemap core and data). Adding the data package as a dependency in the core defeats the purpose of that design.

Ideally that separation should have happen in the `basemap` code itself and not only at packaging time, but `basemap` is under maintenance mode and I doubt someone will implement that.

BTW: I suggest you try `cartopy` for mss and all your mapping needs :wink:

File: recipes/mss/meta.yaml [hunk 26]
```diff
@@ -0,0 +1,85 @@
...
+  build:
+    - basemap ==1.0.7
+    - chameleon >=2.24
+    - curl >=7.45.0
+    - cycler >=0.10.0
+    - execnet >=1.3.0
+    - freetype >=2.5.5
+    - functools32 >=3.2.3.2
+    - hdf5 >=1.8.15.1
+    - libpng >=1.6.22
```
Comment (ocefpaf) [inline recipes/mss/meta.yaml]: Having `>=` is better than `==` b/c it is less strict, but I believe not all of those are true. Also, `libpng`, `hdf5 (and a few others) needs special pinnings too ensure consistent in our stack (meaning that a newer version with a different `ABI` won't break the package). Here `libpng` should be:

`libpng >=1.6.21,<1.7`

File: recipes/mss/meta.yaml [hunk 27]
```diff
@@ -0,0 +1,85 @@
...
+      - mswms = mslib.mswms.mswms:main
+
+requirements:
+  build:
+    - basemap ==1.0.7
+    - chameleon >=2.24
+    - curl >=7.45.0
+    - cycler >=0.10.0
+    - execnet >=1.3.0
+    - freetype >=2.5.5
```
Comment (ocefpaf) [inline recipes/mss/meta.yaml]: Freetype should be:

`freetype 2.6.*`

File: recipes/mss/meta.yaml [hunk 28]
```diff
@@ -0,0 +1,85 @@
...
+    - functools32 >=3.2.3.2
+    - hdf5 >=1.8.15.1
+    - libpng >=1.6.22
+    - lxml >=3.6.0
+    - matplotlib >=1.5.1
+    - netcdf4 >=1.2.2
+    - numpy >=1.11.1
```
Comment (ocefpaf) [inline recipes/mss/meta.yaml]: I believe you don't need to restrict `numpy` to `>=1.11.1`

File: recipes/mss/meta.yaml [hunk 29]
```diff
@@ -0,0 +1,85 @@
...
+requirements:
+  build:
+    - basemap ==1.0.7
+    - chameleon >=2.24
+    - curl >=7.45.0
+    - cycler >=0.10.0
+    - execnet >=1.3.0
+    - freetype >=2.5.5
+    - functools32 >=3.2.3.2
+    - hdf5 >=1.8.15.1
```
Comment (ocefpaf) [inline recipes/mss/meta.yaml]: and `hdf5` should be:

`hdf5 1.8.17|1.8.17.*`

File: recipes/mss/meta.yaml [hunk 30]
```diff
@@ -0,0 +1,85 @@
...
+    - libpng >=1.6.22
+    - lxml >=3.6.0
+    - matplotlib >=1.5.1
+    - netcdf4 >=1.2.2
```
Comment (ocefpaf) [inline recipes/mss/meta.yaml]: Some of your dependencies are over specified b/c `netcdf4` will install `hdf5`, so there is no need to add `hdf5` to the recipe.

File: recipes/mss/meta.yaml [hunk 31]
```diff
@@ -0,0 +1,85 @@
...
+    - numpy >=1.11.1
+    - paste >=1.7.5.1
+    - pillow >=3.2.0
+    - pyqt >=4.11.4,<5
+    - python >=2.7.12,<3
```
Comment (ocefpaf) [inline recipes/mss/meta.yaml]: Is `Mission Support System`  Python 2 only? You don't have to restrict Python here (See my comment above in the build section.)

File: recipes/mss/meta.yaml [hunk 32]
```diff
@@ -0,0 +1,85 @@
...
+
+source:
+    fn: mss-{{ version }}.tar.gz
+    url: https://bitbucket.org/wxmetvis/mss/get/{{ version }}.tar.gz
+    sha256: 6e15a2a7c65ce676169551660e51ced5b0718ef83c4f93104a0f6b70ed502183
+
+build:
+    number: 0
+    script: python setup.py install
+    entry_points:
```
Comment (ocefpaf) [inline recipes/mss/meta.yaml]: If `Mission Support System` is Python 2 only you have to add `skip: True  # [py3k]` here in the build section.

**Comment by patricksnape** [comment]
Closing as per @ocefpaf new PR #1026
